src/components/Situation/Header/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as S from "./styled";
+import IconArrow from "@assets/icons/icon-arrow.svg";
+import IconHome from "@assets/icons/icon-home.svg";
+import { Link, useNavigate } from "react-router-dom";
+import PAGE_PATH from "../../../constants/path";
+
+const Header = () => {
+  const navigate = useNavigate();
+
+  const handleClickArrowIcon = () => {
+    handleGoBack();
+  };
+
+  const handleGoBack = () => {
+    navigate(-1);
+  };
```

Comment:
```suggestion
  const handleRouterBackButtonClick = () => {
    navigate(-1);
  };
```
오호 저는 이렇게 하나로 사용하는 걸 생각했는데 한번 더 추상화하면 어떤 이점이 있나용??

Reply:
@wo-o29 
1. ```navigate(-1);```이 뒤로 가기라는 걸 명시.
2. 새로운 로직이 추가될 때 용이하다?
인데 프로젝트가 작아서 간결하게 가도 될 거 같아요 :)